README_DOCKER.md

```diff
@@ -0,0 +1,68 @@
+# Foodsoft on Docker
+
+This document explains setup and using docker the foodsoft with docker.
+
+## Requirements
+
+* Docker (=> 1.5)
+* Docker Compose (=> 1.1)
+* Nothing more, no ruby, mysql, redis etc!
+
+For installing instructions see https://docs.docker.com/installation/.
+Docker runs on every modern linux kernel, but also with a little help on MacOS
+and Windows!
+
+## Setup
+
+Create docker volume for mysql data:
+
+    mkdir -p ~/.docker-volumes/foodsoft/mysql
+
+Setup foodsoft development data: (This will take some time, containers needs
+to be pulled from docker registry and a lot dependencies needs to be installed.)
+
+    docker-compose run app rake foodsoft:setup_development
+
+TODO: Right know this is not gonna work because of the new database
+configuration via ENV variable. See Notes below.
```

Comment:
What about skipping the database setup step when DATABASE_URL is defined?
(And perhaps the Dockerfile might even remove it before starting, to make local a development install work with Docker as well.)